File: src/main/java/audio_player/MessageTypes.java
package audio_player;

/**
 * Enums for message types <br>
 * Regular message → <i>{@code EG_MESSAGE}</i><br>
 * Embed message → <i>{@code EMBED_MESSAGE}</i>
 *
 * @author krystof-cejchan
 * @since 02/04/22
 */
public enum MessageTypes {

    REG_MESSAGE, EMBED_MESSAGE;

    public MessageTypes[] getAll() {
        return MessageTypes.values();
    }

}

File: src/main/java/enums/MessageTypes.kt
package enums

/**
 * Enums for message types <br></br>
 * Regular message → *`EG_MESSAGE`*<br></br>
 * Embed message → *`EMBED_MESSAGE`*
 *
 * @author krystof-cejchan
 * @since 02/04/22
 */
enum class MessageTypes {
    REG_MESSAGE, EMBED_MESSAGE;

    val all: Array<MessageTypes>
        get() = values()
}